File: nodejs/7 kyu/144.js
// https://www.codewars.com/kata/54e9554c92ad5650fe00022b/train/javascript

// Passed

function toCurrency(price) {
  price = String(price);
  let counter = 0;
  let currency = "";
  for (let idx = price.length - 1; idx >= 0; idx--) {
    counter++;
    if (counter === 4) {
      currency = "," + currency;
      counter = 1;
    }

    currency = price[idx] + currency;
  }

  return currency;
}

const output = toCurrency(123456789012);
console.log(output);

File: python/7 kyu/144.py
# https://www.codewars.com/kata/56069d0c4af7f633910000d3/train/python

# Passed

def search(budget, prices):
    affordable_prices = [price for price in prices if price <= budget]
    affordable_prices.sort()
    affordable_prices_as_string = ",".join([str(price) for price in affordable_prices])
    return affordable_prices_as_string

output = search(14, [13, 15, 14, 14, 15, 13])
print(output)